src/System.Device.Gpio/System/Device/I2c/Drivers/Windows10I2cDevice.Windows.cs

```diff
@@ -23,46 +30,71 @@ public Windows10I2cDevice(I2cConnectionSettings settings)
             DeviceInformationCollection deviceInformationCollection = DeviceInformation.FindAllAsync(deviceSelector).WaitForCompletion();
             if (deviceInformationCollection.Count == 0)
             {
-                throw new ArgumentException($"No I2C device exists for BusId {settings.BusId}", $"{nameof(settings)}.{nameof(settings.BusId)}");
+                throw new ArgumentException($"No I2C device exists for bus ID {settings.BusId}.", $"{nameof(settings)}.{nameof(settings.BusId)}");
             }
 
-            _winDevice = WinI2c.I2cDevice.FromIdAsync(deviceInformationCollection[0].Id, winSettings).WaitForCompletion();
-            if (_winDevice == null)
+            _winI2cDevice = WinI2c.I2cDevice.FromIdAsync(deviceInformationCollection[0].Id, winSettings).WaitForCompletion();
+            if (_winI2cDevice == null)
             {
                 throw new PlatformNotSupportedException($"I2C devices are not supported.");
             }
         }
 
+        /// <summary>
+        /// The connection settings of a device on an I2C bus.

```

Comment:
It looks lke that is what you did anyway, but we should make sure that the comments for all of these implementations of the i2cDevice match.